twingate/twingate_sweeper_test.go

```diff
@@ -26,7 +27,13 @@ func sharedClient(tenant string) (*Client, error) {
 		return nil, fmt.Errorf("must provide environment variable TWINGATE_URL")
 	}
 
-	client := NewClient(os.Getenv("TWINGATE_URL"), os.Getenv("TWINGATE_API_TOKEN"), os.Getenv("TWINGATE_NETWORK"), "sweeper")
+	client := NewClient(
+		os.Getenv("TWINGATE_URL"),
+		os.Getenv("TWINGATE_API_TOKEN"),
+		os.Getenv("TWINGATE_NETWORK"),
+		30*time.Second,
```

Comment:
Lets use 
```golang
value = os.getenv("TWINGATE_HTTP_TIMEOUT", 30*time.Second)
```

Give the option to override that from outside the test 
same for the second variable

Reply:
There is no builtin support for default value when getting env vars but I have added it